onnxruntime/contrib_ops/cuda/layer_norm_impl.cu

```diff
@@ -376,7 +424,7 @@ void HostApplyLayerNorm(
 
 LAYERNORM_LINEAR_IMPL(float, float)
 LAYERNORM_LINEAR_IMPL(half, float)
-LAYERNORM_LINEAR_IMPL(double, float)
+// LAYERNORM_LINEAR_IMPL(double, float)
```

Comment:
do we really support double?
My personal opinion is that we don't need to worry about double type kernel...

Reply:
Do we want to remove the support for double? This is a forward kernel, so might impact inference scenario and will be a breaking change.